electron/plugins/i18n.js

```diff
@@ -10,6 +10,8 @@ import it from './i18n/locales/it.json' with { type: 'json' }
 import ja from './i18n/locales/ja.json' with { type: 'json' }
 import ko from './i18n/locales/ko.json' with { type: 'json' }
 import pl from './i18n/locales/pl.json' with { type: 'json' }
+// eslint-disable-next-line camelcase
```

Comment:
@ivangomes Here and later:
if there's a problem with ESLInt, please change similar imports to `import ptBr`.
But keep file names as `pt_BR`.
And don't forget to remove this ESLint string.